cost/aws/native_utilization/aws_native_utilization.pt

```diff
@@ -0,0 +1,676 @@
+name "AWS Native Utilization"
```

Comment:
Not sure if `Native` is the best way to explain this Policy Template.
AWS CloudWatch Instance Utilization?
AWS Instance Utilization?
...?